packages/fx-core/src/component/generator/copilotPlugin/helper.ts

```diff
@@ -1027,3 +1028,12 @@ export async function updateForCustomApi(
   // 4. update code
   await updateCodeForCustomApi(specItems, language, destinationPath, openapiSpecFileName, needAuth);
 }
+
+const EnvNameMapping: { [authType: string]: string } = {
+  apiKey: "REGISTRATION_ID",
+  oauth2: "CONFIGURATION_ID",
```

Comment:
if you want to use different postfix for the registration id, you should also update the logic in spec-parser/manifestUpdater.ts

Reply:
Using configuration id since we need to align with TDP. Will update spec parser in another pr.